src/PrintEncrypted.cpp

```diff
@@ -0,0 +1,240 @@
+/* A simple program that demonstrates a simple homomorphic program on 4 input values.
+ * */
+
+#include <iostream>
+#include <NTL/ZZ.h>
+#include <NTL/BasicThreadPool.h>
+#include "FHE.h"
+#include "timing.h"
+#include "EncryptedArray.h"
+#include <NTL/lzz_pXFactoring.h>
+
+#include <cassert>
+#include <cstdio>
+
+namespace std {} using namespace std;
+
+int main(int argc, char *argv[])
+{
+  ArgMapping amap;
+
+  // Technical parameter. Use if you want to explicitly set generator elements
+  // for the plaintext arrays.
+  Vec<long> gens;
```

Comment:
in the future just use the long, more descriptive variable name. 

`Vec<long> generators` etc...

yeah, its not as succinct but makes groking it easier for the other guy

( also feel free to bust me for it too :) )

Reply:
Good suggestion. I copied over name defaults from their test files, and swapping over to readable names seems better practice.